### DISCUSSION TITLE
pycaret models do not show up in mlflow ui

### DISCUSSION THREAD

**Discussion by macklark** [pycaret/pycaret#2303] [Q&A] [answered]
A simple pycaret example, but this happens

```python
from pycaret.datasets import get_data

data = get_data('diamond')

import mlflow
mlflow.set_tracking_uri('http://localhost:5000')
from pycaret.regression import *

s = setup(data, target='Price', transform_target=True, log_experiment=True, experiment_name='diamond', silent=True)

best = compare_models()
```

I run a mlflow tracking server `mlflow server --default-artifact-root . --backend-store-uri postgresql://some-postgres-link` and when I open the ui at `http://localhost:5000` this happens

![mlflow window](https://kughclpmuvxmlieyobhv.supabase.in/storage/v1/object/public/images/Capture.PNG)

models do not log to the ui

**Comment by macklark** [comment]
anyone there

**Comment by ngupta23** [comment, marked as answer]
You need to click on the + sign to expand the tracking.

**Reply by macklark** [reply]
Silly of me that worked thank you
